roles/ora-host/tasks/main.yml

```diff
@@ -178,6 +209,31 @@
     - "{{ sysctl_entries }}"
   tags: sysctl
 
+- name: Get the ID of the oinstall group
```

Comment:
The oinstall gid is actually defined in roles/common/defaults/main.yml would it be cleaner to get it there instead of shelling out?

Reply:
I think it would not be cleaner because oracle_groups that is defined in roles/common/defaults/main.yml file is a list containing a bunch of dictionaries. I would have to use a loop and a conditional to find the right item (oinstall group) in the list in order to reference it here. 

As an alternative I could, of course, reference the right item by its index, e.g.: {{ oracle_groups[0].gid }}. This is less flexible and assumes that oinstall is always the first item in the list. The looping or conditional method is more robust if the order of the list might change.  if a separate task is required solely to obtain the group's value, it doesn't significantly differ from simply using the shelling out method.

Reply:
Using https://www.redhat.com/sysadmin/ansible-jinja-lists-dictionaries as an example, how about something like (untested) "{{ oracle_groups | selectattr('group', 'match', 'oinstall') | map(attribute='gid') }}" ?

Reply:
Great idea. It hadn't occurred to me to use a Jinja2 expression for extracting GID. 

Filed https://github.com/google/bms-toolkit/pull/164 for this.